CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ option( TRACCC_SETUP_THRUST
    "Set up the Thrust target(s) explicitly" TRUE )
 option( TRACCC_USE_SYSTEM_THRUST
```

Comment:
Shouldn't this be `TRUE` by default instead?

Reply:
Yes you're absolutely right sorry.